### PR TITLE
Fixes #321: Added missing use of pywbem_os_setup scripts in install

### LINKED ISSUE #321: Docs: pywbem_os_setup missing in installation
The installation section does not mention that the download and use of pywbem_os_setup.sh/.bat is required.

### PR DESCRIPTION
**Pull request by andy-maier**
For details, see the commit message.
Ready for review and merge.

### CODE REVIEW

**Comment by andy-maier** [comment]
Needs to wait for the final solution to PR #277.

File: docs/introduction.rst [hunk 1]
```diff
@@ -118,51 +118,72 @@ development and test environment for pywbem. That has a larger set of OS-level
...
+manager for your Linux distribution (e.g. ``yum`` on the RedHat family), so
+the current userid needs to have sudo permission.
+
+Note that for the time being, the pywbem_os_setup.sh script must be downloaded
+from the pywbemtools project. See
+`issue #329 <https://github.com/pywbem/pywbemtools/issues/329>`_.
+
+On native Windows, download the ``pywbem_os_setup.bat`` script and run it to
+install the needed OS-level packages:
+
```
Comment (KSchopmeyer) [inline docs/introduction.rst]: how do you get wget.  We are back to chocolatey again

File: docs/introduction.rst [hunk 2]
```diff
@@ -183,7 +204,7 @@ branch of the Git repository of the package:
 .. _`Verification of the installation`:
 
 Verification of the installation
-""""""""""""""""""""""""""""""""
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can verify that the pywbemtools package and its dependent packages are
 installed correctly by invoking pywbemcli. Invoking with the ``--version``
```
Comment (KSchopmeyer) [inline docs/introduction.rst]: Lets add that --version returns the current versions of pywbemtools and pywbem.  For example:

File: docs/introduction.rst (hunk 1 above)
Comment (andy-maier) [reply docs/introduction.rst]: wget is stated as a requirement for all platforms. IMO, that is sufficient, i.e. we don't need to explain how to use the package manager of each platform (including native Windows) to get it.

File: docs/introduction.rst (hunk 2 above)
Comment (andy-maier) [reply docs/introduction.rst]: That's already in the text.

**Pull request merged**